kroxylicious-systemtests/src/main/resources/helm_vault_overrides.yaml

```diff
@@ -6,6 +6,8 @@
 
 # Helm Overrides File for HashiCorp Vault used by the system tests.
 server:
+  image:
+    tag: 1.18.3
```

Comment:
https://developer.hashicorp.com/vault/docs/platform/k8s/helm/configuration#tag-2

Reply:
Should we remove the helm chart version in [Vault.java](https://github.com/kroxylicious/kroxylicious/blob/e925c2ad28f80241497a0e6455159a1ba1c45be0/kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/vault/Vault.java#L109)? Not sure if it is going to work without it